exp-player/addon/mixins/full-screen.js

```diff
@@ -39,7 +39,7 @@ export default Ember.Mixin.create({
 
     actions: {
         showFullscreen: function (elementId) {
-            elementId = elementId || this.get('fullScreenElementId');
+            elementId = this.get('fullScreenElementId');
```

Comment:
If we're ignoring any value passed in, can I suggest removing the function parameter (now unused) and changing this line to `var elementId`?